docs/admin/admin-ui/faq.md

```diff
@@ -0,0 +1,63 @@
+# Frequently Asked Questions (FAQ)
+
+
+### Why is the Gluu Flex Admin UI displaying the following error messages after the Flex VM installation?
+
+
+#### The requested page not found
+
+```
+Error Code: 404
+The requested page was not found on this server.
+```
+
+If a user encounters the above error when visiting the Admin UI URL, it indicates that the Admin UI is not properly installed. Please verify whether the Admin UI build is located at `/var/www/html/admin`. If the build is not present at this location, Janssen displays this error.
+
+####  Admin UI backend is down
+
+```
+Error Code: 503
+Gluu Flex Admin UI is not getting any response from the backend (Jans Config Api).
+```
+
+Gluu Flex Admin UI facilitates interaction with the Jans Auth Server through a REST API layer, [Jans Config API](https://docs.jans.io/v1.0.16/contribute/implementation-design/jans-config-api/). This error prompts administrators to perform a series of troubleshooting steps.
```

Comment:
You should use [version placeholder](https://docs.jans.io/head/CONTRIBUTING/#referencing-janssen-project-release-in-documents) when using an full URL.

Reply:
done